desktop/ui/src/main/java/org/datacleaner/windows/AnalysisJobBuilderWindowImpl.java

```diff
@@ -702,18 +703,32 @@ private void addComponentDescriptorButtons(JToolBar toolBar) {
             final String description = "<html><b>" + name + "</b><br/>" + superCategory.getDescription() + "</html>";
 
             final PopupButton popupButton = new PopupButton(name);
+
             applyMenuPopupButttonStyling(popupButton);
 
             DCPopupBubble popupBubble = new DCPopupBubble(_glassPane, description, 0, 0,
                     IconUtils.getComponentSuperCategoryIcon(superCategory));
-            popupBubble.attachTo(popupButton);
+            
+            popupBubble.attachTo(popupButton, new DCPopupBubble.PopupCallback() {
+                @Override
+                public boolean onBeforeShow() {
+                    for(PopupButton scButton : _superCategoryButtons) {
+                        if(scButton.isSelected()){
+                            return false;
+                        }
+                    }
+                    return true;
+                }
+            });
 
             final JPopupMenu menu = popupButton.getMenu();
 
             final DescriptorMenuBuilder menuBuilder = new DescriptorMenuBuilder(_analysisJobBuilder, _usageLogger,
                     superCategory, null);
             menuBuilder.addItemsToPopupMenu(menu);
-
+            if (menu.isVisible()) {
```

Comment:
Is this needed? Seems that this will never evaluate to true since the menu is being built at this point in time.